### PR TITLE
Fix UI of clear search button

### LINKED ISSUE #134: Improve the UI of "Clean Search" button
**Is your feature request related to a problem? Please describe.**
The "Clear Search" button becomes visible when the user search for something. By clicking it, the search results are removed from the local Storage and image grid. 

**Present UI**
- a small cross enclosed inside a red square.
- Both the color and the position is odd.

74013862-66269680-49b3-11ea-99e6-cec6ae68fbb1

**Describe the solution you'd like**
This one is tricky. We have a very limited space available. 
- The first thought is to add an icon alongside the filter icon when there are images in the grid. But, most users would assume it to be the icon that will clear the text they have written in the search box.
- A better solution can be to add a button of absolute position in bottom left (?)

Better ideas are welcomed.

### PR DESCRIPTION
**Pull request by AmanR28**
Fixes #134 

**Description**

Improve UI of clear search button.

**Other information**

Screenshot from 2020-02-13 21-40-48

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Comment by AmanR28** [comment]
@makkoncept Please suggest the position of clear search button.

**Comment by makkoncept** [comment]
The icon does not blends with other icons. Why not use the [font awesome times icon](https://fontawesome.com/v4.7.0/icon/times). Also, I think we can experiment by placing the icon inside the search bar (on the left of the filter icon).

**Comment by makkoncept** [comment]
@AmanR28 what's the status?

**Comment by AmanR28** [comment]
@makkoncept Is this alright?

**Comment by makkoncept** [comment]
@AmanR28 Please paste the screenshot of the change

**Comment by AmanR28** [comment]
Screenshot from 2020-02-14 23-48-48

File: src/popup/popup.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
           <input type="text" placeholder="search the commons..." id="section-search-input" autofocus />
           <i class="fa fa-search" id="search-icon"></i>
           <i class="fa fa-filter" id="filter-icon"></i>
-          <button class="clear-search-button">X</button>
+          <i class="fa fa-times-circle clear-search-button" style="color: #ff4e14; right: 12%"></i>
```
Comment (makkoncept) [inline src/popup/popup.html]: don't add inline css. Define it in `_search.scss`

**Review by makkoncept — CHANGES_REQUESTED** [review]
Please fix this
Screenshot from 2020-02-15 00-05-57

File: src/popup/popup.html (hunk 1 above)
Comment (AmanR28) [reply src/popup/popup.html]: I first tried that, but it is getting overwritten. Can I use !important?

Comment (makkoncept) [reply src/popup/popup.html]: Okay. It will be fixed soon.

**Review by makkoncept — APPROVED** [review]
Works fine. Merging.

**Pull request merged**